internal/test/e2e/databasesql/traces.json

```diff
@@ -63,6 +63,226 @@
+              "parentSpanId": "xxxxx",
+              "spanId": "xxxxx",
+              "status": {},
+              "traceId": "xxxxx"
+            },
+            {
+              "attributes": [
+                {
+                  "key": "db.query.text",
+                  "value": {
+                    "stringValue": "DROP TABLE contacts"
+                  }
+                },
+                {
+                  "key": "db.operation.name",
+                  "value": {
+                    "stringValue": "DROP TABLE"
+                  }
+                },
+                {
+                  "key": "db.collection.name",
+                  "value": {
+                    "stringValue": "contacts"
+                  }
+                }
+              ],
+              "flags": 256,
+              "kind": 3,
+              "name": "DROP TABLE contacts",
+              "parentSpanId": "xxxxx",
+              "spanId": "xxxxx",
+              "status": {},
+              "traceId": "xxxxx"
+            },
+            {
+              "attributes": [
+                {
+                  "key": "db.query.text",
+                  "value": {
+                    "stringValue": "CREATE TABLE foo (Bar int)"
+                  }
+                },
+                {
+                  "key": "db.operation.name",
+                  "value": {
+                    "stringValue": "CREATE TABLE"
+                  }
+                },
+                {
+                  "key": "db.collection.name",
+                  "value": {
+                    "stringValue": "foo"
+                  }
+                }
+              ],
+              "flags": 256,
+              "kind": 3,
+              "name": "CREATE TABLE foo",
+              "parentSpanId": "xxxxx",
+              "spanId": "xxxxx",
+              "status": {},
+              "traceId": "xxxxx"
+            },
+            {
+              "attributes": [
+                {
+                  "key": "db.query.text",
+                  "value": {
+                    "stringValue": "ALTER TABLE foo ADD baz int"
+                  }
+                },
+                {
+                  "key": "db.operation.name",
+                  "value": {
+                    "stringValue": "ALTER TABLE"
+                  }
+                },
+                {
+                  "key": "db.collection.name",
+                  "value": {
+                    "stringValue": "foo"
+                  }
+                }
+              ],
+              "flags": 256,
+              "kind": 3,
+              "name": "ALTER TABLE foo",
+              "parentSpanId": "xxxxx",
+              "spanId": "xxxxx",
+              "status": {},
+              "traceId": "xxxxx"
+            },
+            {
+              "attributes": [
+                {
+                  "key": "db.query.text",
+                  "value": {
+                    "stringValue": "syntax error"
```

Comment:
will follow up to parse error response codes from databases separately, and mark them as error spans appropriately. Don't want to grow this PR too much and this is equivalent to what we do now